.github/actions/setup/action.yml

```diff
@@ -15,3 +15,7 @@ runs:
       run: npm ci
       shell: bash
       if: steps.cache.outputs.cache-hit != 'true'
+    - name: Install Foundry
+      uses: foundry-rs/foundry-toolchain@v1
+      with:
+        version: nightly
```

Comment:
Why is nightly needed?

Reply:
I don't know, I copied it from the foundry tests job.